src/openforms/logging/templates/logging/events/outgoing_request_log_details_view_admin.txt

```diff
@@ -0,0 +1,4 @@
+{% load i18n %}
+{% blocktrans trimmed with lead=log.fmt_lead user=log.fmt_user timelinelog=log.pk %}
+   {{ lead }}: User {{ user }} viewed outgoing request log {{ timelinelog }} in the admin
```

Comment:
seeing a primary key of a log record that is periodically pruned is not very useful :)

Please include:

* host
* path
* http method

of the request as the representation of the log record.